components/nexus-security/src/main/java/org/apache/shiro/nexus5727/FixedDefaultWebSessionManager.java

```diff
@@ -36,15 +36,20 @@
 public class FixedDefaultWebSessionManager
     extends DefaultWebSessionManager
 {
+  /**
+   * Default HTTP session cookie name that Nexus uses - NXSESSIONID
+   */
+  public static final String DEFAULT_NEXUS_SESSION_COOKIE_NAME = "NXSESSIONID";
```

Comment:
I am unaware where this could break people - maybe old ajp connectors had JSESSIONID hardcoded back in the day?. FWIW CLM recently changed their cookie id from JSESSIONID to CLMSESSIONID and no one has complained so far. That said the install base is much smaller.

Regardless it is still configurable so one can put it back to the old name if they needed too.